fedimint-cli/src/lib.rs

```diff
@@ -395,6 +395,10 @@ Examples:
         /// Which server to send request to
         #[clap(long = "peer-id")]
         peer_id: Option<u16>,
+        /// Guardian password in case authenticated API endpoints are being
+        /// called. Only use together with --peer-id.
+        #[clap(long = "auth", requires = "peer_id")]
```

Comment:
The format of auth is not clear to me. And example would be great to have, possibly on the whole command (we already have some on `DevCmd`).

Reply:
Is it `user:pass`?

Reply:
There is no user, so I thought it would be self-explanatory (Guardian password=what you set on federation setup). But I actually noticed we already have an optional password flag at the top level that we could use here.

Reply:
Oh, OK. BTw. Made me realize that in real settings people will probably quickly want user + password and some permission system so they can more granularily work in their organizations etc.